src/base/utils/version.h

```diff
@@ -55,6 +53,11 @@ namespace Utils
 
         constexpr Version() = default;
 
+        Version(const QStringView string)
```

Comment:
Having both `Version(const QStringView string)` and `fromString()` is redundant.

Reply:
fromString can take a default value and I don't think it is suitable for Version constructor.
Or I can add a TODO for now to refactor later.